nautobot/docs/configuration/optional-settings.md

```diff
@@ -348,6 +351,17 @@ LOGGING = {
 
 ---
 
+## LOGIN_URL
+
+Default: `"/login/"`
+
+The URL where requests are redirected for login to authenticate users requiring access to protected views. Any views inheriting from [`LoginRequiredMixin`](https://docs.djangoproject.com/en/stable/topics/auth/default/#django.contrib.auth.mixins.LoginRequiredMixin) or [`AccessMixin`](https://docs.djangoproject.com/en/stable/topics/auth/default/#django.contrib.auth.mixins.AccessMixin) will require login.
+
+!!! warning
+    When setting [`BASE_PATH`](#base_path) you must also update [`LOGIN_URL`](#login_url) to begin with the value of `BASE_PATH`. For example if `BASE_PATH` is set to `"nautobot/"`, then `LOGIN_URL` must be set to `"/nautobot/login/"`.
+
```

Comment:
Maybe also add a note about required LOGIN_URL value if SOCIAL_AUTH_ENABLED? Ref nautobot/core/cli.py:

```python
    if settings.SOCIAL_AUTH_ENABLED:
        settings.INSTALLED_APPS.append("social_django")
        settings.AUTHENTICATION_BACKENDS.insert(0, settings.SOCIAL_AUTH_MODULE)
        backend_name = get_sso_backend_name(settings.SOCIAL_AUTH_MODULE)
        settings.LOGIN_URL = "/{}login/{}/".format(settings.BASE_PATH, backend_name)
```

It'd be really slick if the "hint" message you added in the error check could also include this intelligence.

Reply:
Yeah that's a good call, thanks.